apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/MainActivity.java

```diff
@@ -126,6 +126,7 @@ protected void onCreate(Bundle savedInstanceState) {
         if (!TextUtils.isEmpty(apiUrl)) {
             Distribute.setApiUrl(apiUrl);
         }
+        //noinspection deprecation
```

Comment:
New line before this one.

Reply:
@guperrot Should we do it with `noinspection` comments? I don't see a single approach to this in the project code

Reply:
Our convention is to have a newline before every comment.  We've had conversations about why this is/isn't a good rule before, but that's what we've chosen to do for the time being.

Reply:
Looks like the convention in this repo is to *NOT* add a newline before compiler warning suppressions.

Reply:
I'm aware of that. My question is: Does this rule apply to the service marks for warnings suppressing

Reply:
We add it when we don't forget but it was missed during some reviews, especially on unit tests. I don't usually nitpick on those `//` ones but if you can do the change that would be good I guess.

Reply:
Will do